xmr-btc/src/monero.rs

```diff
@@ -97,6 +97,14 @@ impl Sub for Amount {
     }
 }
 
+impl Mul for Amount {
+    type Output = Amount;
+
+    fn mul(self, rhs: Self) -> Self::Output {
+        Self(self.0 * rhs.0)
+    }
+}
+
```

Comment:
```suggestion
impl Mul<i32> for Amount {
    type Output = Amount;

    fn mul(self, rhs: i32) -> Self::Output {
        Self(self.0 * rhs)
    }
}

```

Multiplying amounts does not make sense.

Reply:
Edit: it should be u64, not i32.

Reply:
Why doesn't it make sense? This trait is implementing `Monero::Amount * Monero::Amount`